### ISSUE TITLE
Labeling Errors in Graph Visualizations

### ISSUE THREAD

**Issue by laceyg** [dmlc/xgboost#6278]
There appears to be errors in some graph visualizations, which do not agree with the text representation of the xgb trees. In the following code snippet, you can see the disagreement between the yes/no/missing labels and the visual representation (e.g. notice how 'no' and 'yes' are flipped).

![image](https://user-images.githubusercontent.com/5554069/96911041-4ca87a80-146e-11eb-8ade-27cb53090993.png)

![image](https://user-images.githubusercontent.com/5554069/96910955-34386000-146e-11eb-82c4-5068a7a32431.png)

```
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt
import xgboost as xgb

np.random.seed(0)
targ_var = 'A'
size = 1000
dataset = pd.DataFrame(np.random.randint(0, 100, size=(size, 2)).astype(np.float32)
                       , columns=['B','C'])
dataset[targ_var] = (np.random.randint(0, 2, size=(size, 1)).astype(np.int32))
p_vars = list(dataset.columns.drop(targ_var))

params = {
    'base_score': 0.5  
    , 'gamma': 0
    , 'learning_rate': 0.3
    , 'max_depth': 3
    , 'min_child_weight': 1
    , 'n_estimators': 5
    , 'nthread': 24 
    , 'objective': 'binary:logistic'
    , 'random_state': 0
    , 'reg_alpha': 0
    , 'reg_lambda': 0 
    , 'scale_pos_weight': 1
    , 'subsample': 1
    , 'verbosity': 1
    , 'eval_metric': 'logloss'
    , 'tree_method': 'hist'
}
bst_params = {**params}

model = xgb.XGBClassifier(**bst_params)
model.fit(dataset[p_vars], dataset[targ_var])
bstr = model.get_booster()
trees = bstr.get_dump('', with_stats=True)

tree_i=0
print(trees[tree_i])
xgb.to_graphviz(model, num_trees=tree_i, rankdir='LR')
```
This script seems to reproduce these errors in versions 1.0.2, 1.1, 1.2, and the latest 1.3 nightly I have tried.
